ow_plexil/src/plans/FaultHandlingPattern2.plp

```diff
@@ -8,47 +8,54 @@
 // If an antenna fault but no arm faults occurring, only action 2 occurs.
```

Comment:
I did not find this to be the case in testing. If I only inject a antenna tilt locked fault, then action 1 continues.

Reply:
I have fixed the incorrect comment.  This plan responds to PAN faults only, not tilt faults, which do not affect panning.  Good catch!